raiden/tests/utils/geth.py

```diff
@@ -94,7 +94,7 @@ def geth_to_cmd(
     cmd.extend([
         '--nodiscover',
         '--rpc',
-        '--rpcapi', 'eth,net,web3',
+        '--rpcapi', 'eth,net,web3,personal',
```

Comment:
used by the smoketest, we could add this as part of the `GethNodeDescription` in the future.